### ISSUE TITLE
Can we no longer get an API key?

### ISSUE THREAD

**Issue by Neeha-DataScience** [ZeevG/python-forecast.io#67]
I am trying to run a basic Python example from your GitHub repo for which I need an API key. Your website says "We are no longer accepting new signups." Does this mean it is no longer possible to obtain an API key?

**Comment by wcdawn** [comment]
Yes. See #66. The API will function through the end of the year for those with an API key but no more keys are being issued.
